ubuntu-latest/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM ubuntu:latest
+LABEL version="1"
+ARG ARCH
+
+RUN DEBIAN_FRONTEND=noninteractive TZ=Europe/Zurich && \
+    ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && \
+    echo $TZ > /etc/timezone && \
+    apt-get update -qq && \
+    apt-get dist-upgrade -y && \
+    apt-get install -y apt-utils wget \
+                       autoconf \
+                       automake \
+                       astyle \
+                       clang \
+                       cmake \
+                       docker.io \
+                       gcc \
+                       g++ \
+                       git \
+                       libtool \
+                       libssl-dev \
+                       libunwind-dev \
+                       make \
+                       ninja-build \
+                       pkg-config \
+                       python3 \
+                       python3-nose \
+                       python3-rednose \
+                       python3-pytest \
+                       python3-pytest-xdist \
+                       python3-psutil \
+                       python3-pip \
+                       zlib1g-dev \
+                       maven \
+                       openjdk-11-jdk \
+                       qemu-system \
+                       valgrind \
+                       flex bison \
+                       lsb-release software-properties-common \
+                       lcov \
+                       graphviz \
+                       golang-go \
+                       yamllint \
+                       python3-tabulate python3-jinja2
+
+RUN cd / && git clone --depth 1 --branch Release_1_10_0 https://github.com/doxygen/doxygen.git && cd doxygen && mkdir build && cd build && cmake -G "Unix Makefiles" .. && make -j 4 && make install && doxygen --version 
```

Comment:
Can we please comment them out for now? I'd like to see how/that OQS CI works OK with vanilla noble.

Reply:
Seems many packages we installed are not being used, so I just installed packages required for building liboqs and OQS-BoringSSL.

Reply:
Well the image(s) are meant to be used by all OQS sub projects and they have additional needs...

Reply:
Yes. They may add them to the apt install command later. I'm trying to minimize the building time :)